arch/riscv/Kconfig

```diff
@@ -199,6 +199,25 @@ config RISCV_GENERIC_TOOLCHAIN
 	  Allow SOCs that have custom extended riscv ISA to still
 	  compile with generic riscv32 toolchain.
 
+config RISCV_KERNEL_IN_USER_MODE
```

Comment:
Would love to help revive this PR. This particular set of configs *seems* like it should be wrapped in a `choice`.